server/datastore/mysql/migrations/tables/20241116233322_AddLuksDataToHostDiskEncryptionKeys.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241116233322, Down_20241116233322)
+}
+
+func Up_20241116233322(tx *sql.Tx) error {
+	_, err := tx.Exec(`ALTER TABLE host_disk_encryption_keys
+    		ADD COLUMN base64_encrypted_salt VARCHAR(255) NOT NULL DEFAULT '' AFTER base64_encrypted,
```

Comment:
`[...] CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci`

Reply:
Touché. Fixing.

Reply:
Of note, we could save storage here by keeping the charset as-is, because the column should never be joined or sorted, and since this is base64'd even ASCII would represent this fine. But might as well be consistent.